src/main/java/org/zalando/fahrschein/domain/Subscription.java

```diff
@@ -21,24 +19,34 @@
     @Nullable
     private final OffsetDateTime createdAt;
 
-    @JsonCreator
-    @Hack("Use String type for createdAt so we do not have to require JavaTimeModule to be registered in the ObjectMapper")
-    public Subscription(@JsonProperty("id") String id, @JsonProperty("owning_application") String owningApplication, @JsonProperty("event_types") Set<String> eventTypes, @JsonProperty("consumer_group") String consumerGroup, @JsonProperty("created_at") String createdAt) {
-        this(id, owningApplication, eventTypes, consumerGroup, createdAt == null ? null : OffsetDateTime.parse(createdAt));
-    }
-
-    public Subscription(String owningApplication, Set<String> eventTypes, String consumerGroup) {
-        this(null, owningApplication, eventTypes, consumerGroup, (String)null);
+    @Hack("Necessary to enable Jackson Wrapper working with multiple constructors but without @JsonCreator annotation")
```

Comment:
Would marking the constructor (the one with all parameters) as `JsonCreator` also work? That sounds less like a hack to me.